hubblestack/extmods/fileserver/azurefs.py

```diff
@@ -168,7 +165,7 @@ def serve_file(load, fnd):
     with hubblestack.utils.files.fopen(fpath, 'rb') as fp_:
         fp_.seek(load['loc'])
         data = fp_.read(__opts__['file_buffer_size'])
-        if data and six.PY3 and not salt.utils.is_bin_file(fpath):
+        if data and not salt.utils.is_bin_file(fpath):
```

Comment:
In the code we are moving, every salt dependency should be moved. Unless it is a high-level module like loader, for which we already took tasks.

This dependency has also to be moved to our code. "salt.utils.is_bin_file"